randomizer/Lists/CustomLocations.py

```diff
@@ -701,6 +709,18 @@ class LocationTypes(IntEnum):
             logic=lambda l: (l.grape and l.lanky) or l.phasewalk,
             group=4,
         ),
+        CustomLocation(
+            map=Maps.AztecLlamaTemple,
+            name="In the Water",
+            x=2273,
+            y=207,
+            z=2385,
+            max_size=64,
+            logic_region=Regions.LlamaTemple,
+            logic=lambda l: l.HasGun(Kongs.any) and (l.istiny and l.settings.krusha_kong == Kongs.tiny) or (not l.istiny) and l.swim and l.scope and Events.AztecLlamaSpit in l.Events,
```

Comment:
Without testing. This can mean, as long as you aren't tiny, but have swim and sniper and the Llama has spitted, you can break the crate. 
Or it means what you intended it to mean. It's not very readable if the reader has to guess.